From 3d19c3c8ad4f2371893eb478523369a2ad8f67b3 Mon Sep 17 00:00:00 2001
From: Yi Chen <yichen@qti.qualcomm.com>
Date: Fri, 28 Aug 2015 19:24:45 +0800
Subject: qcacld: Fix compiling errors when BUILD_DEBUG_VERSION := 0.

This is a compiling issue when BUILD_DEBUG_VERSION := 0 is set in Kbuild.

Change-Id: Iaee98eb91d66990dfa4989a7c07cb5592f8a6079
CRs-Fixed: 898366
---
 CORE/UTILS/FWLOG/dbglog_host.c | 4 ++--
 Kbuild                         | 9 +++++----
 2 files changed, 7 insertions(+), 6 deletions(-)

diff --git a/CORE/UTILS/FWLOG/dbglog_host.c b/CORE/UTILS/FWLOG/dbglog_host.c
index e45a49a..96b4402 100644
--- a/CORE/UTILS/FWLOG/dbglog_host.c
+++ b/CORE/UTILS/FWLOG/dbglog_host.c
@@ -55,8 +55,6 @@
 #include <a_debug.h>
 #define FWLOG_DEBUG   ATH_DEBUG_MAKE_MODULE_MASK(0)
 
-#if defined(DEBUG)
-
 static bool appstarted = FALSE;
 static bool senddriverstatus = FALSE;
 static bool kd_nl_init = FALSE;
@@ -64,6 +62,8 @@ static int cnss_diag_pid = INVALID_PID;
 static int get_version = 0;
 static int gprint_limiter = 0;
 
+#if defined(DEBUG)
+
 static ATH_DEBUG_MASK_DESCRIPTION g_fwlogDebugDescription[] = {
     {FWLOG_DEBUG,"fwlog"},
 };
diff --git a/Kbuild b/Kbuild
index ff51a9c..798efe2 100644
--- a/Kbuild
+++ b/Kbuild
@@ -966,11 +966,12 @@ CDEFINES += -DWCN_PRONTO
 CDEFINES += -DWCN_PRONTO_V1
 endif
 
-ifeq ($(BUILD_DEBUG_VERSION),1)
-CDEFINES +=	-DWLAN_DEBUG \
-		-DTRACE_RECORD \
+CDEFINES += 	-DTRACE_RECORD \
 		-DLIM_TRACE_RECORD \
-		-DSME_TRACE_RECORD \
+		-DSME_TRACE_RECORD
+
+ifeq ($(BUILD_DEBUG_VERSION),1)
+CDEFINES +=	-DWLAN_DEBUG \
 		-DPE_DEBUG_LOGW \
 		-DPE_DEBUG_LOGE \
 		-DDEBUG
-- 
cgit v1.1

